src/Components/IndexScene/InterceptBanner.tsx

```diff
@@ -24,21 +24,21 @@ export function InterceptBanner(props: { onRemove: () => void }) {
           , or see{' '}
           <a
             className={'external-link'}
```

Comment:
I know this is a nit, but can we change this to just `"eternal-link"` please?

Reply:
Yup, agreed and fixed